### PR TITLE
Monke Eth on Base

### PR DESCRIPTION
**Pull request by TeflonYian**
**Description**

A clear and concise description of the features you're adding in this pull request.

**Tests**

Please describe any tests you've added. If you've added no tests, or left important behavior untested, please explain why not.

**Additional context**

Add any other context about the problem you're solving.

**Metadata**

- Fixes #[Link to Issue]

### CODE REVIEW

**Comment by TeflonYian** [comment]
IMG_3257_adobe_express (1)

File: data/MONKE/logo.svg
Comment (TeflonYian) [inline data/MONKE/logo.svg:1]: IMG_3257_adobe_express (1)

**Review by cfluke-cb — APPROVED** [review]
Looks good on Base side.

**Review by annieke — DISMISSED** [review]
can you make sure the logo renders correctly? the current HTML file is malformed

**Comment by TeflonYian** [comment]
IMG_20240329_230452_483

**Comment by TeflonYian** [comment]
IMG_3257_adobe_express (2)

**Comment by TeflonYian** [comment]
resized-10789efd

**Comment by cfluke-cb** [comment]
Hi @TeflonYian the image is still invalid. Can you update with a working png or svg?

**Comment by TeflonYian** [comment]
resized-9d8bcdb1

**Comment by TeflonYian** [comment]
> Hi @TeflonYian the image is still invalid. Can you update with a working png or svg?

320574714-2beebcb9-820f-4fa6-a69f-1eaff05ef128

**Comment by TeflonYian** [comment]
monke_svg

**Comment by TeflonYian** [comment]
Please approve this works

**Review by wbnns — APPROVED** [review]
ACK

**Review by annieke — APPROVED** [review]

**Pull request merged**